### PR TITLE
Allowed text method to pass on multiline_text method specific arguments

### LINKED ISSUE #1645: Provide additional keyword args for ImageDraw.text(...) method for multiline text
The check for ``if self._multiline_check(text):`` inside of the ``ImageDraw.text(...)`` method [1] is great, but if True, it goes to the next line ``return self.multiline_text(xy, text, fill, font, anchor)`` and doesn't allow the user to pass important keyword args to the ``multiline_text(...)`` function like **align** and **spacing**.  Instead, why not do something like:

```py
def text(self, xy, text, fill=None, font=None, anchor=None, **kwargs):
    """:param kwargs: additional args to pass to multiline_text if the text is multiline"""
    if self._multiline_check(text):
        return self.multiline_text(xy, text, fill, font, anchor, **kwargs)
```

Then I can call the ``.text(...)`` function with those additional params:

```py
draw = ImageDraw.Draw(myImage)
draw.text((0,0), 'hello world', align='left', spacing=5)
```

This would be a passive change to the current ``.text(...)`` method call.

[1] https://github.com/python-pillow/Pillow/blob/master/PIL/ImageDraw.py#L245

### PR DESCRIPTION
**Pull request by radarhere**
Resolves #1645

### CODE REVIEW

**Comment by troygrosfield** [comment]
:+1:

**Comment by wiredfool** [comment]
Can you add in the new parameters to `docs/reference/ImageDraw.rst`?

**Comment by radarhere** [comment]
Okay, I've added documentation. Let me know if you have any thoughts. I wasn't quite sure whether to specify the additional arguments individually, or to just reference `args` and `kwargs`.

File: Tests/test_imagefont.py [hunk 1]
```diff
@@ -121,6 +121,7 @@ def test_textsize_equal(self):
             size = draw.textsize(txt, ttf)
             draw.text((10, 10), txt, font=ttf)
             draw.rectangle((10, 10, 10 + size[0], 10 + size[1]))
+            del draw
```
Comment (wiredfool) [inline Tests/test_imagefont.py]: Why are you adding `del draw` to all the tests?

Comment (radarhere) [inline Tests/test_imagefont.py]: My understanding is that `del draw` performs a cleanup operation to save memory. It was present on some tests, but not others. If you'd like to go the other way, and remove it in order to reduce unnecessary lines of code in the test suite, just say the word.

Comment (wiredfool) [inline Tests/test_imagefont.py]: Just wondering, as that seems to be an unrelated change in the test suite.
